iceoryx_hoofs/include/iceoryx_hoofs/log/logstream.hpp

```diff
@@ -18,6 +18,7 @@
 #ifndef IOX_HOOFS_LOG_LOGSTREAM_HPP
 #define IOX_HOOFS_LOG_LOGSTREAM_HPP
 
+#include "iceoryx_hoofs/cxx/string.hpp"
```

Comment:
Can this be removed? The logger is used in `string.inl` which would result in cyclic dependencies